public/app/plugins/datasource/grafana-azure-monitor-datasource/dashboards/vMInsightsRG.json

```diff
@@ -0,0 +1,1904 @@
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "Subscriptions",
+        "multi": false,
+        "name": "sub",
+        "options": [],
+        "query": "Subscriptions()",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": "${ds}",
+        "definition": "ResourceGroups($sub)",
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "Resource Group",
+        "multi": true,
+        "name": "rg",
+        "options": [],
+        "query": "ResourceGroups($sub)",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "allValue": null,
+        "current": {
+          "selected": true,
+          "text": "Average",
+          "value": "score = round(avg(Val), 2)"
+        },
+        "description": null,
+        "error": null,
+        "hide": 0,
+        "includeAll": false,
+        "label": "Aggregate",
+        "multi": false,
+        "name": "agg",
+        "options": [
+          {
+            "selected": true,
+            "text": "Average",
+            "value": "score = round(avg(Val), 2)"
+          },
+          {
+            "selected": false,
+            "text": "P5th",
+            "value": "score= round(percentile(Val, 5), 2)"
+          },
+          {
+            "selected": false,
+            "text": "P10th",
+            "value": "score= round(percentile(Val, 10), 2)"
+          },
+          {
+            "selected": false,
+            "text": "P50th",
+            "value": "score= round(percentile(Val, 50), 2)"
+          },
+          {
+            "selected": false,
+            "text": "P80th",
+            "value": "score= round(percentile(Val, 80), 2)"
+          },
+          {
+            "selected": false,
+            "text": "P90th",
+            "value": "score= round(percentile(Val, 90), 2)"
+          },
+          {
+            "selected": false,
+            "text": "P95th",
+            "value": "score= round(percentile(Val, 95), 2)"
+          }
+        ],
+        "query": "Average : score = round(avg(Val)\\, 2), P5th : score= round(percentile(Val\\, 5)\\, 2),  P10th : score= round(percentile(Val\\, 10)\\, 2), P50th : score= round(percentile(Val\\, 50)\\, 2), P80th : score= round(percentile(Val\\, 80)\\, 2), P90th : score= round(percentile(Val\\, 90)\\, 2), P95th : score= round(percentile(Val\\, 95)\\, 2)",
+        "queryValue": "",
+        "skipUrlSync": false,
+        "type": "custom"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-15m",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Azure VM Monitoring - VM Insights by Resource Group",
+  "uid": "-Nj9uzFnz2",
```

Comment:
again not sure if we want to keep a uid in here, I think it could potentially cause problems?